src/links.js

```diff
@@ -11,826 +11,1127 @@ import { imagePath, hostname, imgify } from "./data/misc";
+			image: imagePath + '/execom/pes/pragya_singh.png',
+		},
+		{
+			name: 'Nitin S',
+			position: 'Treasurer',
+			image: imagePath + '/execom/pes/nitin_s.jpg',
+		},
+	],
+	sps: [
+		{
+			name: 'Prof. K Nagamani',
+			position: 'Faculty Advisor',
+			image: imagePath + '/execom/sps/nagamani_maam.jpg',
+		},
+		{
+			name: 'Chirag C Rao ',
+			position: 'Chair',
+			image: imagePath + '/execom/sps/chirag_c_rao.jpg',
+		},
+		{
+			name: 'Mustafa Chikhliwala',
+			position: 'Secretary',
+			image: imagePath + '/execom/sps/mustafa_chikhliwala.jpg',
+		},
+		{
+			name: 'Rajeev Singh ',
+			position: 'Treasurer',
+			image: imagePath + '/execom/sps/rajeev_singh.jpg',
+		},
+	],
+	wie: [
+		{
+			name: 'Prof Deepika',
+			position: 'Faculty Advisor',
+			image: imagePath + '/execom/wie/deepika_maam.jpg',
+		},
+		{
+			name: 'Jahnavi M Math',
+			position: 'Chair',
+			image: imagePath + '/execom/wie/jahnavi_m_math.jpg',
+		},
+		{
+			name: 'Omisha Singh',
+			position: 'Secretary',
+			image: imagePath + '/execom/wie/omisha_singh.jpg',
+		},
+		{
+			name: 'Gohitha Maheshwari J',
+			position: 'Treasurer',
+			image: imagePath + '/execom/wie/gohitha_maheshwari.jpg',
+		},
+	],
+	ras: [
+		{
+			name: 'Dr. Abhay Deshpande',
+			position: 'Faculty Advisor',
+			image: imagePath + '/execom/ras/abhay_a_deshpande.jpg',
+		},
+		{
+			name: 'Shreya Sakri',
+			position: 'Chair',
+			image: imagePath + '/execom/ras/shreya_sakri.jpeg',
+		},
+		{
+			name: 'S Lingeshwara',
+			position: 'Secretary',
+			image: imagePath + '/execom/ras/s_lingeshwara.jpg',
+		},
+		{
+			name: 'S Vedram',
+			position: 'Treasurer',
+			image: imagePath + '/execom/ras/s_vedram.png',
+		},
+	],
+	sight: [
+		{
+			name: 'Dr Usha J',
+			position: 'Faculty Advisor',
+			image: imagePath + '/execom/sight/usha_maam.jpg',
+		},
+		{
+			name: 'Lalith K',
+			position: 'Chair',
+			image: imagePath + '/execom/sight/lalith_kishore.jpg',
+		},
+		{
+			name: 'Nandan Chebbi',
+			position: 'Secretary',
+			image: imagePath + '/execom/sight/nandan_chebbi.jpg',
+		},
+		{
+			name: 'Neha P',
+			position: 'Treasurer',
+			image: imagePath + '/execom/sight/neha_p.jpg',
+		},
+	],
+	cas: [
+		{
+			name: 'Dr. Shylashree.N ',//needs to be updated
+			position: 'Faculty Advisor',
+			image: imagePath + '/execom/sight/usha_maam.jpg',
+		},
+		{
+			name: 'Akash Kotagi',
+			position: 'Chair',
+			image: imagePath + '/execom/cas/akash_kotagi.jpg',
+		},
+		{
+			name: 'Sanjana Carol',
+			position: 'Secretary',
+			image: imagePath + '/execom/cas/sanjana_carol.jpg',
+		},
+		{
+			name: 'Sai Surya Sreekar Mulukutla',
+			position: 'Treasurer',
+			image: imagePath + '/execom/cas/sreekar_mulukutla.jpg',
+		},
+	],
+	sc: [
+		{
+			name: 'Dr. Veena Divya K',
+			position: 'Faculty Advisor',
+			image: imagePath + '/execom/sensorscouncil/veena_maam.jpg',
+		},
+		{
+			name: 'Rakshan R Kulkarni ',
+			position: 'Chair',
+			image: imagePath + '/execom/sensorscouncil/rakshan_kulkarni.HEIC',
```

Comment:
Here's the reference. Please confirm this is working

Reply:
ahh, sensors council page is not still ready that's why I didn't see any issues 👍🏼

Reply:
It is better to change now then, otherwise we need to make it do it separately later